component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/voiceotp/VoiceOTPConstants.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.authenticator.voiceotp;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+/**
+ * VoiceOTPConstants class for maintain constants of the project.
+ */
+public class VoiceOTPConstants {
+
+    private static final String VOICEOTP_PREFIX = "VoiceOTP-";
+    public static final String AUTHENTICATOR_NAME = "VoiceOTP";
+    public static final String AUTHENTICATOR_FRIENDLY_NAME = "Voice OTP";
+    public static final String ALGORITHM_NAME = "SHA1PRNG";
+    public static final String ALGORITHM_HMAC = "HmacSHA1";
```

Comment:
With this [1] we removed the SHA1 usage. Shall we change here as well ?

[1] https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/173